packages/kbn-test/src/auth/session_manager.ts

```diff
@@ -54,7 +54,12 @@ export class SamlSessionManager {
     this.isCloud = options.isCloud;
     this.log = options.log;
     // if the rolesFilename is provided, respect it. Otherwise use DEFAULT_ROLES_FILE_NAME.
-    const rolesFile = rolesFilename ? rolesFilename : this.DEFAULT_ROLES_FILE_NAME;
+    let rolesFile: string;
```

Comment:
This can probably be simplified altogether to this:
```
let rolesFile = process.env.ROLES_FILENAME_OVERRIDE ?? rolesFilename ?? this.DEFAULT_ROLES_FILE_NAME;
```

Reply:
Restored old functionality in this.